src/core/transactions/ViewVotingKeyLinkTransaction.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 NEM Foundation (https://nem.io)
```

Comment:
The license should not use NEM Foundation anymore. should just use `NEM`
e.g. `Copyright 2018-present NEM`

perhaps, replace all occurrence, I can be a new PR, but okay with changing everything in this one.

Reply:
@yilmazbahadir is on this one